api/tests/Feature/PoolCandidateTest.php

```diff
@@ -545,18 +545,27 @@ public function testFilterByEmploymentEquity(): void
 
     public function testFilterByLanguageAbility(): void
     {
+        $noLanguageUser = User::factory()->create();
+        $noLanguageUser->looking_for_english = false;
```

Comment:
To set intial values, pass an object with key-value pairs to `create`, like on line 555.

Reply:
okay...
done in https://github.com/GCTC-NTGC/gc-digital-talent/pull/6014/commits/661fc939f34e583a40b2311c26d43658e020db8b

Reply:
Oh yeah, I like that much better!